src/zsys.c

```diff
@@ -995,16 +995,8 @@ zsys_udp_new (bool routable)
         return INVALID_SOCKET;
     }
 
-    //  If there's no SOCK_CLOEXEC, let's try the second best option. Note that
-    //  race condition can cause socket not to be closed (if fork happens
-    //  between socket creation and this point).
-#if !defined ZMQ_HAVE_SOCK_CLOEXEC && defined FD_CLOEXEC
-    if (fcntl (udpsock, F_SETFD, FD_CLOEXEC) == SOCKET_ERROR)
-        zsys_socket_error ("fcntl (FD_CLOEXEC)");
-#endif
-
     //  On Windows, preventing sockets to be inherited by child processes.
-#if defined ZMQ_HAVE_WINDOWS && defined HANDLE_FLAG_INHERIT
+#if defined (__WINDOWS__) && defined (HANDLE_FLAG_INHERIT)
```

Comment:
There's CZMQ_HAVE_WINDOWS please use that one

Reply:
Fixed in latest commit